manifests/qpid.pp

```diff
@@ -26,20 +25,4 @@
   }
 
   contain qpid
-
-  qpid::config::queue { $katello::params::candlepin_event_queue:
```

Comment:
Do we need to care about removing this queue?

Reply:
It won't be a problem since it will land in tandem with the artemis piece in Katello

Reply:
But qpid will stick around for Pulp 2, will it be an issue if the queue persists there?

Reply:
Oh misunderstood your question. It'll just be a lonesome empty queue that no one publishes or subscribes to, so I think it's fine